backend/stakepoold/rpcclient.go

```diff
@@ -1,13 +1,16 @@
 package main
 
 import (
+	"encoding/hex"
 	"fmt"
 	"io/ioutil"
+	"sync/atomic"
```

Comment:
Danger, danger!  This almost certainly is abused.  When synchronizing sections you pretty much always want to use a mutex.  atomic should only be used if you are creating your own locking types.